src/services/snapshot-service.ts

```diff
@@ -52,14 +53,12 @@ export default class SnapshotService extends PercyClientService {
   create(
     name: string,
     resources: any[],
-    enableJavaScript = false,
-    widths: number[] | null = null,
-    minHeight: number | null = null,
+    options: SnapshotOptions = {},
     clientInfo: string | null = null,
     environmentInfo: string | null = null,
   ): Promise<any> {
     const snapshotCreationPromise: Promise<any> = this.percyClient.createSnapshot(
-      this.buildId, resources, {name, widths, enableJavaScript, minimumHeight: minHeight, clientInfo, environmentInfo},
+      this.buildId, resources, { name, ...options, minimumHeight: options.minHeight, clientInfo, environmentInfo },
```

Comment:
Heads up that in this implementation we are passing to the client both `minimumHeight` and `minHeight` in the options object. I don't think it matters, though, and this version is cleaner than manually passing through each option just so that we can rename `minHeight` to `minimumHeight`.